spec/rex/powershell/command_spec.rb

```diff
@@ -3,6 +3,10 @@
 
 
 def decompress(code)
+  if code =~ /powershell.exe.*(?:-c|-Command)\s(.*)$/
+    code = Regexp.last_match(1).gsub("''", "'")
+  end
```

Comment:
Detect if the "code" we're decompression is actually a command instead of raw powershell code. If it is a command, extract the powershell code and undo the way single quotes are escaped. This ensures that `code` is valid powershell code when it is passed to the decompression logic.